src/handlers/VehicleTestController.ts

```diff
@@ -148,6 +148,12 @@ export class VehicleTestController implements IVehicleTestController {
       }
       // map testTypes back after validation
       payload.testTypes = testTypes;
+      payload.testTypes.forEach(x=>{
```

Comment:
I think it would probably be better to do. this for loop above where the assigning payload.testTypes to testTypes to keep all of the processing together